docs/howto-use-prometheus-metrics.md

```diff
@@ -1,5 +1,7 @@
 # How to use Prometheus metrics
```

Comment:
I think this should also go to `dev-guides`.

Reply:
I think so too.